int64 sequence_id
int64 sequence_length
string sequence_name
int64 size
CotsDetection[] detection
CotsMaximumScore[] maximum_scores
